invokeai/frontend/web/src/features/lora/components/LoRACard.tsx

```diff
@@ -80,7 +80,7 @@ const LoRAContent = memo(({ lora }: { lora: LoRA }) => {
             onChange={handleChange}
             min={DEFAULT_LORA_WEIGHT_CONFIG.sliderMin}
             max={DEFAULT_LORA_WEIGHT_CONFIG.sliderMax}
-            step={DEFAULT_LORA_WEIGHT_CONFIG.coarseStep}
+            step={DEFAULT_LORA_WEIGHT_CONFIG.fineStep}
```

Comment:
These composite input components have both `step` and `fineStep` props. `fineStep` is used when the user holds `shift`.